src/components/organisms/PageHeader/ProjectSelection.tsx

```diff
@@ -78,6 +81,15 @@ const ProjectSelection = () => {
   };
 
   const handleProjectChange = (project: Project) => {
+    const confirmed = ipcRenderer.sendSync('confirm-action', {
```

Comment:
I imagine this is more than fast enough since the IPC happens on the same computer but my experience with Electron is limited. We can optimise a bit by doing the skip if unsaved resource count equals zero here but I left it lean until performance is an issue.